src/main/target/SKYSTARSH743HD/target.c

```diff
@@ -36,15 +36,19 @@ timerHardware_t timerHardware[] = {
     DEF_TIM(TIM4, CH1, PD12, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 2),   // S3
     DEF_TIM(TIM4, CH2, PD13, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 3),   // S4
 
-    DEF_TIM(TIM5, CH1, PA0, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 4),   // S5  
+#if defined(SKYSTARSH743HD_SERVOS)
```

Comment:
There is an alternative solution. Set S5-S8 as MC_SERVO and then use of flag `output_mode` to `MOTORS` enables the X8 support. But we indeed do not need 2 targets for this board

Reply:
I have updated the branch according to your comment.